lib/events/pgevents/common/utils.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgcommon
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/gravitational/trace"
+	"github.com/jackc/pgerrcode"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/sirupsen/logrus"
+
+	"github.com/gravitational/teleport/api/utils/retryutils"
+)
+
+// TODO(espadolini): delete in favor of lib/backend/pgbk/common
```

Comment:
The same as above. Also looks like all that code will be removed "soon" after it's merged?

Reply:
Either #29025 gets merged first, in which case I'll update this PR to use `lib/backend/pgbk/common`, or this PR gets merged first, in which case I'll update #29025 to delete `lib/events/pgevents/common` in favor of `lib/backend/pgbk/common`. A better name/place to put that is very welcome, tho 😅

Reply:
Moved it to `lib/backend/pgbk/common` now, it makes it easier to keep it in sync between the two PRs anyway.